### ISSUE TITLE
What classes should be excluded for proguard?

### ISSUE THREAD

**Issue by Gintasz** [MachinePublishers/jBrowserDriver#287]
I'm trying to proguard my java app, but I get "could not launch browser" when init() is called. Works fine without proguard.

Proguard basically obfuscates class names and members, but I guess reflection functionality is used, so it messes up.

Anyone knows?

**Comment by hollingsworthd** [comment]
Yeah on obfuscation there are probably some hardcoded package paths. Plus with all the classpath trickery to launch a child process. Reflection is used. Honestly this is lower priority for me. It's an open source project. I welcome PRs to help make this project more compatible with obfuscation but it's not something that will be inherently in this project's interest.

**Comment by hollingsworthd** [comment]
All else being equal this should work so it's worth leaving this issue open.
